code/modules/clothing/head/helmet.dm

```diff
@@ -305,6 +305,28 @@
 /obj/item/clothing/head/helmet/marine/standard
 	flags_item_map_variant = (ITEM_JUNGLE_VARIANT|ITEM_ICE_VARIANT|ITEM_PRISON_VARIANT|ITEM_ICE_PROTECTION)
 
+/obj/item/clothing/head/helmet/marine/standard/attack_self(mob/user)
+
+	///List of colors for the M-10 and their respective icon states
+	var/static/list/helmet_colors = list(
```

Comment:
what? why are you redeclaring the colors everytime.

Put that on the item, not in the proc


@@ -305,6 +305,28 @@
/obj/item/clothing/head/helmet/marine/standard
	flags_item_map_variant = (ITEM_JUNGLE_VARIANT|ITEM_ICE_VARIANT|ITEM_PRISON_VARIANT|ITEM_ICE_PROTECTION)
	var/static/list/helmet_colors = list( bla bla bla...